apps/finance/test/finance.js

```diff
@@ -91,6 +91,7 @@ contract('Finance App', accounts => {
 
     it('sets the end of time correctly', async () => {
         const { financeApp } = await newProxyFinance()
+        await financeApp.mock_setTimestamp(100) // to make sure it overflows
```

Comment:
I'd expand on this comment, to something like `to make sure it overflows with MAX_UINT64 period length`